docker/latest/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="info@dianomic.com"
+
+ENV ARCHITECTURE="x86_64"
+ENV OS="ubuntu1804"
+
+ARG PKG_VERSION="latest"
+
+ENV FLEDGE_PKGS_ARCHIVES_LINK="http://archives.fledge-iot.org/${PKG_VERSION}/${OS}/${ARCHITECTURE}/"
```

Comment:
Do we have WL handling for this?

Reply:
Yes, it is handled in white-labeling scripts.